apparmor.d/groups/virt/containerd

```diff
@@ -72,28 +76,29 @@ profile containerd @{exec_path} flags=(attach_disconnected) {
   @{sys}/module/apparmor/parameters/enabled r,
 
         @{PROC}/@{pid}/task/@{tid}/ns/net rw,
+  owner @{PROC}/@{pids}/attr/current      r,
   owner @{PROC}/@{pids}/uid_map           r,
   owner @{PROC}/@{pids}/mountinfo         r,
         @{PROC}/sys/net/core/somaxconn    r,
 
-  deny /dev/bsg/            rwkl,
-  deny /dev/bus/            rwkl,
-  deny /dev/bus/usb/        rwkl,
-  deny /dev/bus/usb/[0-9]*/ rwkl,
-  deny /dev/char/           rwkl,
-  deny /dev/cpu/            rwkl,
-  deny /dev/cpu/[0-9]*/     rwkl,
-  deny /dev/dma_heap/       rwkl,
-  deny /dev/dri/            rwkl,
-  deny /dev/dri/by-path/    rwkl,
-  deny /dev/hugepages/      rwkl,
-  deny /dev/input/          rwkl,
-  deny /dev/input/by-id/    rwkl,
-  deny /dev/input/by-path/  rwkl,
-  deny /dev/net/            rwkl,
-  deny /dev/snd/            rwkl,
-  deny /dev/snd/by-path/    rwkl,
-  deny /dev/vfio/           rwkl,
+  /dev/bsg/            r,
+  /dev/bus/            r,
+  /dev/char/           r,
+  /dev/cpu/            r,
+  /dev/cpu/[0-9]*/     r,
+  /dev/dma_heap/       r,
+  /dev/dri/            r,
+  /dev/dri/by-path/    r,
+  /dev/hugepages/      r,
+  /dev/input/          r,
+  /dev/input/by-id/    r,
+  /dev/input/by-path/  r,
+  /dev/net/            r,
+  /dev/snd/            r,
+  /dev/snd/by-path/    r,
+  /dev/vfio/           r,
+
+  deny / r,
```

Comment:
I would recommend not denying `/` on a profile that has the `attach_disconnected` flag as `/` can be the root of something else than the host.

Reply:
Good point, I removed it.